src/backend/joanie/core/migrations/0040_alter_order_payment_schedule_alter_order_state.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.13 on 2024-06-27 09:11
+
+from django.db import migrations, models
+import joanie.core.fields.schedule
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0039_alter_order_has_consent_to_terms_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='payment_schedule',
```

Comment:
Why this field is updated here ?

Reply:
[done properly on dev branch](https://github.com/openfun/joanie/pull/825/commits/bd6e35e4d997874d1a61977854bb978a15712b2f)